source/examples/rapids-sagemaker-hpo/notebook.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !docker build -t $ecr_fullname .
</pre><p> </p><p>Passing <code>--build-arg RAPIDS_IMAGE</code> doesn't do anything here, because earlier cells re-write the <code>Dockerfile</code> and don't include handling for that argument (they just directly template in the relevant <code>rapidsai/core</code> image in the top-level <code>FROM</code>).     </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/381/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>